pymc/distributions/continuous.py

```diff
@@ -1641,7 +1641,20 @@ def logp(value, b, kappa, mu):
             kappa > 0,
             msg="b > 0, kappa > 0",
         )
-
+    def icdf(value, b, kappa, mu):
+        if b <= 0:
+            raise ValueError("Scale Parameter b must be positive")
+        if value <= 0.5:
+            res = mu - (b / kappa) * pt.log(2 * value)
+        if value > 0.5:
```

Comment:
Ah, comparisons need to be done using PyTensor operations

Again, there may be other edits that need to be incorporated, but a robust way to go about this is to check via a unit test, hence the emphasis for testing

Reply:
@larryshamalama I changed the comparisons to Pytensor Operations